opennlp-tools/src/main/java/opennlp/tools/dictionary/Dictionary.java

```diff
@@ -313,9 +306,9 @@ public boolean hasNext() {
           }
 
           public String next() {
-            return entries.next().getStringList().getToken(0);
+            return String.join(" ",entries.next().getStringList());
```

Comment:
Given the javadoc comment on line 297 this may be the desired behavior but we'll need to double-check that.